Makefile

```diff
@@ -41,16 +41,34 @@ metalint: | $(METALINT)
 lint:
 	$(LINT) $$($(PKGS)) | grep -v -E "exported (function)|(var)|(method)|(type).*should have comment or be unexported"
 
+#Test
+TEST_DB = vulcanize_testing
+TEST_CONNECT_STRING = postgresql://localhost:5432/$(TEST_DB)?sslmode=disable
+
 .PHONY: test
 test: | $(GINKGO) $(LINT)
 	go vet ./...
 	go fmt ./...
+	dropdb --if-exists $(TEST_DB)
+	createdb $(TEST_DB)
+	cd db/migrations;\
+    	$(GOOSE) postgres "$(TEST_CONNECT_STRING)" up
+	cd db/migrations/;\
```

Comment:
Any chance this works without these `cd` operations?

Reply:
iirc the `cd` was a fix for the `-dir` flag not working as expected with `goose`, but I think in the end it was me being stupid: https://github.com/pressly/goose/issues/139

Reply:
Short answer: yes, if you do it right :sweat_smile: